hack/machineset.sh

```diff
@@ -184,8 +223,22 @@ get_vsphere_ms() {
   local provider=$2
 
   # set golden image template name
-  # TODO: read from parameter
-  template="windows-golden-images/windows-server-2004-template"
+  template=""
+  case "${WIN_SRV,,}" in
+    2022)
+      template="windows-golden-images/windows-server-2022-template"
+    ;;
+    2019)
+      echo "Note: Windows Server ${WIN_SRV} is not supported for platform ${provider}"
+      template="windows-golden-images/vm-winsrv-1909-golden-image"
+    ;;
+    20h2)
+      template="jvaldes/windows-server-20h2-template"
```

Comment:
templates should be in the folder `windows-golden-images`, otherwise the pruner will delete them

Reply:
The only 20h2 template we have is one you created under `jvaldes` folder. Given 20H2 going out of support in May, I'd rather not get too hung up on this as it's only for dev testing.

Reply:
If there is no `permanent` template available, I suggest we just drop 20h2 from this PR, considering the focus on 2022 moving forward